code/game/objects/structures/reagent_dispensers.dm

```diff
@@ -209,6 +209,12 @@
 /obj/structure/reagent_dispensers/fueltank/attackby(obj/item/W as obj, mob/user as mob)
 	src.add_fingerprint(user)
 
+	if(HAS_TRAIT(W, TRAIT_TOOL_WRENCH))
+		if(user.action_busy)
+			return TRUE
+		toggle_anchored(W, user)
+		return TRUE
```

Comment:
the code was more or less redundent and I decided to just revert to calling the parent. All future reagent_dispenser children will need to have wrenchable specified, otherwise they just default to being wrenchable. Objects that shouldn't be wrenchable have been marked as so in this update